interactions/client/bot.py

```diff
@@ -156,126 +182,113 @@ async def __compare_sync(self, data: dict, pool: List[dict]) -> bool:
+                                                    break
+
+                                                for choice in option.get("choices"):
+                                                    for _choice in _option.get("choices"):
+                                                        if choice["name"] == _choice["name"]:
+                                                            for choice_attr in choice_attrs:
+                                                                if (
+                                                                    choice.get(choice_attr)
+                                                                    and not _choice.get(choice_attr)
+                                                                    or not choice.get(choice_attr)
+                                                                    and _choice.get(choice_attr)
+                                                                ):
+                                                                    clean = False
+                                                                    break
+                                                                elif choice.get(
+                                                                    choice_attr
+                                                                ) != _choice.get(choice_attr):
+                                                                    clean = False
+                                                                    break
+                                                                else:
+                                                                    continue
+                                                            if not clean:
+                                                                break
+                                                        if not clean:
+                                                            break
+                                                    if not clean:
+                                                        break
+                                            elif option.get(option_attr) != _option.get(
+                                                option_attr
+                                            ):
+                                                clean = False
+                                                break
+                                            else:
+                                                continue
+                                        if not clean:
+                                            break
+                                    if not clean:
+                                        break
+                                if not clean:
+                                    break
+
+                        if not clean:
+                            break
+
+                        else:
+                            continue
```

Comment:
I'm usually fine with hacky code, especially in Python. Obviously, it's not *great*, but sometimes it's the only way of doing something without resorting to something arguably worse.

that being said wtf there *has* to be a better way

Reply:
I just realized something lol

Reply:
fixed